cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java

```diff
@@ -83,6 +88,7 @@ public String storeSystemConfig(@Validated @ModelAttribute SystemConfigPage syst
             model.addAttribute("show_alert", true);
             model.addAttribute("alert_type", "danger");
             model.addAttribute("alert_msg", "An unexpected error occurred. System Configuration could not be saved.");
+            messageHandler.handleMessage("An unexpected error occurred. System Configuration could not be saved.", e, MessageType.ERROR);
```

Comment:
This message should be more specific about what happened and not the message for the users.